src/components/calcite-dropdown/calcite-dropdown.e2e.ts

```diff
@@ -990,6 +990,7 @@ describe("calcite-dropdown", () => {
         const wrapper = document.querySelector(wrapperName);
         wrapper.shadowRoot.querySelector<HTMLElement>("#item-3").click();
         await new Promise<void>((resolve) => requestAnimationFrame(() => resolve()));
+        await new Promise<void>((resolve) => requestAnimationFrame(() => resolve()));
```

Comment:
Weird. Does the test hang if you don't add this delay?

Reply:
Yeah it won't pass.

Reply:
It says item-1 instead of item-3 if this isn't there.

Reply:
Wonder if something else is going on. Not sure why the changes would cause this to fail. Can you open a low-prio issue for me to look into this?